incubator/hnc/pkg/reconcilers/hnc_config_test.go

```diff
@@ -37,6 +51,7 @@ var _ = Describe("HNCConfiguration", func() {
 		Eventually(func() error {
 			return resetHNCConfigToDefault(ctx)
 		}).Should(Succeed())
+		SetDefaultEventuallyTimeout(2 * time.Second)
```

Comment:
Is there a way to restore this to what it used to be (e.g. in case it wasn't 2s for some reason)?

Reply:
Ditto. Resolve this comment since it is outdated.